python/ray/tests/test_metrics.py

```diff
@@ -112,6 +112,38 @@ def f(self):
                     or "travis" in process)
         break
 
+    # Test kill_actor.
+    def actor_killed(pid):
+        """ Check For the existence of a unix pid. """
```

Comment:
Can you remove the spaces at the beginning and end of the string and also replace `pid` with `PID`?